Makefile

```diff
@@ -2,35 +2,59 @@
 
 PYTHON3?=python3
 
+UV := $(shell command -v uv 2> /dev/null)
```

Comment:
Suddenly wondering whether this works on Windows or not.
@Geobert can you test this, please?

Reply:
will do this evening :)